<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationLogs</class>
 <widget class="QDialog" name="OperationLogs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
       </property>
       <property name="text">
        <string>快速查询：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="quickSearch">
       <item>
        <property name="text">
         <string>今天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>昨天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>最近七天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>本月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>上个月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>自定义</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
       </property>
       <property name="text">
        <string>开始时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="startDateTimeEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
       </property>
       <property name="text">
        <string>结束时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="endDateTimeEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
       </property>
       <property name="text">
        <string>设备选择：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item>
      <widget class="QPushButton" name="getDataBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton {
width:112px;
height:34px;
background:rgba(81,223,0,1);
border:0px;
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(255,255,255,1);
}</string>
       </property>
       <property name="text">
        <string>数据导出</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
